custom/icds_reports/static/icds_reports/js/spec/prevalence-of-severe-report.directive.spec.js

```diff
@@ -21,7 +14,7 @@ describe('Prevalence Of Severe Directive feature flag disable', function () {
     pageData.registerUrl('icds_locations', 'icds_locations');
 
     beforeEach(module('icdsApp', function ($provide) {
-        provideConstants($provide);
+        utils.provideDefaultConstants($provide, true, true);
```

Comment:
I don't love the way this function call looks (like that there's no `kwargs` to give you a hint what the booleans do). but decided it was fine for the use case.